packages/edit-site/src/hooks/push-changes-to-global-styles/index.js

```diff
@@ -323,6 +323,7 @@ function PushChangesToGlobalStylesControl( {
 
 	return (
 		<BaseControl
+			__nextHasNoMarginBottom
```

Comment:
## Testing instructions

In the Site Editor, open a page with a Heading block, or add one. In the block inspector, expanded the Advanced section to see the "Apply globally" control at the bottom.

<img src="https://github.com/user-attachments/assets/59aff777-c4e9-4020-b9b7-afb0c3b54863" alt="Apply globally control" width="274">